### ISSUE TITLE
ERROR: Error occured in image_search: image_search() Blocked

### ISSUE THREAD

**Issue by tek0011** [tek0011/reddit_info_bot#87]
This is currently blocking the bot from starting.  It just crashes over and over.  

```
2016-03-24 07:52:12 [reddit_info_bot.reddit] INFO: [N] Detected keyword(s) u/info_bot_nsfw in https://www.reddit.com/r/info_bot_nsfw/comments/4br8b9/httpiimgurcomlfz9zipjpg/d1bn32j
2016-03-24 07:52:12 [reddit_info_bot.reddit] ERROR: Error occured in image_search: image_search() 
takes exactly 1 argument (2 given)
```

**Comment by nyov** [comment]
Fixed, I reset the git clone on the server.

**Comment by tek0011** [comment]
tyvm good sir.

On Thu, Mar 24, 2016 at 9:44 AM, nyov <notifications@github.com> wrote:

> Closed #87 <https://github.com/tek0011/reddit_info_bot/issues/87>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tek0011/reddit_info_bot/issues/87#event-601837973>
>



-- 
Jeff
